irods_testing_environment/irods_config.py

```diff
@@ -10,21 +10,58 @@
 from . import execute
 from . import json_utils
 
+irods_zone = dict()
+
+irods_version = dict()
```

Comment:
Consider adding some comments explaining what these variables are.

Are these variables meant to be used outside of this file or are they private?

Reply:
They aren't meant for use outside of this file. To be honest, I'm not sure how to prevent that. I just needed them to outlive the function calls in which they are used. :)

Reply:
Done, I think